src/routes/api/collabs/[id]/picks/[pick_id]/index.ts

```diff
@@ -153,6 +199,10 @@ export const del: RequestHandler = async ({ request, params }) => {
 	const decodedUser = Jwt.decode(cookies['user_id']);
 	const userId = decodedUser['user_id'] as string;
 
+	const reason = await request.json().then(function (data) {
```

Comment:
This syntax is very unnecessary. `(await request.json()).Reason` achieves the same effect.

Reply:
Oh and I suggest keeping reason lowercase, looks just a bit neater, but not a biggie.

Reply:
Oh ok! I made reason lowercase